tools/invoke-tests

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+# process arguments
+if [[ $1 == --coverage ]]; then
+    WITH_COVER_OPTIONS=1
+    shift
+fi
+if [[ $# -lt 3 ]]; then
+    echo "usage: $0 [--coverage] prove-tool-path make-tool-path build-directory [test-file...]"
+    exit
+fi
+prove_path=$1 make_path=$2 build_dir=$3
+shift 3
+test_files=("$@")
+
+# set Perl module include path and coverage options
+export PERL5LIB="$PWD:$PWD/ppmclibs:$PWD/ppmclibs/blib/arch/auto/tinycv:$PERL5LIB"
```

Comment:
I assume you want this to be either the root source dir in case of autotools-make and relative to the out-of-source cmake build dir but `$PWD` would only match by chance, right?
I guess it's a safer approach to check explicitly for CMAKE_CURRENT_BINARY_DIR or so and fallback to `$(dirname "$0")/..`

Reply:
I thought of CMAKE_CURRENT_BINARY_DIR because I assumed with cmake we would build ppmclibs below the build dir

Reply:
I see, then it's alright